doc/src/design/ir.md

```diff
@@ -232,8 +244,10 @@ containing the correctly serialized transactions `v15` and `v30`.
 | `SendGetCFilters`| Sends a `getcfilters` message. |
 | `SendGetCFHeaders`| Sends a `getcfheaders` message. |
 | `SendGetCFCheckpt`| Sends a `getcfcheckpt` message. |
+| `SendCompactBlock` | Sends a `cmpctblock` message. |
 | **Other** | |
 | `Nop` | No operation. Used during minimization. |
+| `Probe` | Tells the scenario to record the incoming messages from the peer. |
```

Comment:
```suggestion
| `Probe` | Tells the scenario to probe state for the fuzzer (e.g. received messages, tip hash, ...). |
```

Since we are going to do more than just recording incoming messages